schemas/complementary_currencies-v1.json

```diff
@@ -1,10 +1,13 @@
 {
   "$schema": "https://json-schema.org/draft-04/schema#",
-  "id": "https://cdn.murmurations.tech/schemas/complementary_currencies-v1.json",
+  "id": "https://some.url/from.matt",
```

Comment:
@matslats Please revert back to the CDN link. We need an ID with a working URL or the validator will not work properly. Thanks.

Reply:
Never mind this request. I will adjust it when I make another change to deal with the issue about multi-select we have been discussing.